examples/build.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
```

Comment:
Think we can add some documentation to specify exactly what this build file is?  This is just a template right?